src/python/pants/engine/internals/parser.py

```diff
@@ -90,6 +91,7 @@ def parse(
         # those extra symbols will not see our target aliases etc. This also means that if multiple
         # prelude files are present, they probably cannot see each others' symbols. We may choose
         # to change this at some point.
+
```

Comment:
Nit, would be good to remove this because now the comment is separated from the code.